weave/ops_domain/artifact_version_ops.py

```diff
@@ -544,21 +545,33 @@ def history_metrics(
 
 # TODO: Move all this to helper functions off the artifactVersion object
 def _artifact_version_to_wb_artifact(artifactVersion: wdt.ArtifactVersion):
-    entity_name = artifactVersion["artifactSequence"]["defaultArtifactType"]["project"][
-        "entity"
-    ]["name"]
-    project_name = artifactVersion["artifactSequence"]["defaultArtifactType"][
-        "project"
-    ]["name"]
+    print(f"\n\nlogging version query result: ===> \n{artifactVersion}\n\n", flush=True)
+    artifact_id = artifactVersion["id"]
     type_name = artifactVersion["artifactSequence"]["defaultArtifactType"]["name"]
     home_sequence_name = artifactVersion["artifactSequence"]["name"]
     commit_hash = artifactVersion["commitHash"]
+    uri = artifact_wandb.WeaveWBArtifactByIDURI(
+        home_sequence_name, commit_hash, artifact_id
+    )
+    print(f"\n\nlogging artifact_by_id uri: ===> \n{uri}\n\n", flush=True)
+    # todo: postman query returns null project for user querying an artifact from a team they're
```

Comment:
is this still a puzzle? Def seems odd

Reply:
I can remove that comment actually. When I updated my Weave_wandb_cookie to a non-admin api key, everything worked as expected.